cheevos/cheevos.c

```diff
@@ -2955,16 +2955,21 @@ static int cheevos_iterate(coro_t *coro)
          *************************************************************************/
    CORO_SUB(LYNX_MD5)
 
-      if (coro->len < 0x0240)
+      /* Checks for the existence of a headered Lynx file.
+         Unheadered files fall back to GENERIC_MD5. */
+
+      const int LYNX_HEADER_LEN = 0x40;
+
+      if (coro->len <= LYNX_HEADER_LEN ||
+        memcmp("LYNX", (void *)coro->data, 5) != 0)
```

Comment:
`5` is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming))?

Reply:
Literals in C are guaranteed by the standard to end with a null terminator. The Lynx ROM header also has a 0 byte after `LYNX`, so `memcmp` is comparing the four letters and the null terminator in one go.

Reply:
Sounds good, thanks :+1: